lib/validation-stream.js

```diff
@@ -9,6 +9,12 @@ var Duplex = require('readable-stream').Duplex;
 
 var XmlParser = require('./xml-parser');
 
+/** @arg {?string} str Input string to check.
+    @return {!boolean} True if input is undefined, null, or entirely whitespace. */
+function isBlank(str) {
```

Comment:
Yep, I typically prefer to only break out code when reuse is necessary or the function is getting too big and state is getting hard to track/manage.

One downside to unit testing for something like this is it's typically overkill as you would need to not only test the method itself but also have some spy-esque assertion to verify we are using `isBlank` in the code, at the appropriate time, with the expected content (aka a contract).